### DISCUSSION TITLE
Dynamically generated content and `content` configuration

### DISCUSSION THREAD

**Discussion by sebastiansulinski** [tailwindlabs/tailwindcss#6374] [Help]
Most of our site's structure comes from database and with v3 I cannot seem to be able to find the option to include all components and classes and only disable the ones I don't want as `content` property is required in order to generate any output.

Is there any way to achieve this?

**Comment by nik-lampe** [comment]
Having the same issue.

It would be great to just be able to provide an async function to the `content` property which then retrieves the contents from anywhere.
